src/components/views/EditMilestone.jsx

```diff
@@ -746,13 +749,13 @@ class EditMilestone extends Component {
                       <Toggle
                         id="itemize-state"
                         defaultChecked={this.state.itemizeState}
-                        onChange={() => this.toggleItemize()}
+                        onChange={this.toggleItemize}
                         disabled={!isNew && !isProposed}
                       />
                       <span className="label">Add multiple expenses, invoices or items</span>
                     </div>
 
-                    {!itemizeState && (
+                    {!itemizeState ? (
```

Comment:
This is alright but unnecessary modification.

Reply:
Agree, but it prevents to render nullable element.

Reply:
👍